secant/secant-testnet-Info.plist

```diff
@@ -67,6 +67,24 @@
 		<string>Inter-ExtraBold.otf</string>
 		<string>Inter-ExtraLightItalic.otf</string>
 		<string>Inter-ExtraLight.otf</string>
+        <string>Archivo-Thin.otf</string>
+        <string>Archivo-ThinItalic.otf</string>
+        <string>Archivo-BlackItalic.otf</string>
+        <string>Archivo-Black.otf</string>
+        <string>Archivo-BoldItalic.otf</string>
+        <string>Archivo-Bold.otf</string>
+        <string>Archivo-MediumItalic.otf</string>
+        <string>Archivo-Medium.otf</string>
+        <string>Archivo-Italic.otf</string>
+        <string>Archivo-Regular.otf</string>
+        <string>Archivo-LightItalic.otf</string>
+        <string>Archivo-Light.otf</string>
+        <string>Archivo-SemiBoldItalic.otf</string>
+        <string>Archivo-SemiBold.otf</string>
+        <string>Archivo-ExtraBoldItalic.otf</string>
+        <string>Archivo-ExtraBold.otf</string>
+        <string>Archivo-ExtraLightItalic.otf</string>
+        <string>Archivo-ExtraLight.otf</string>
```

Comment:
Nonblocking nit: do you want the indentation to be consistent here?

Reply:
Strange, the info file is modified on a visual representation level

<img width="548" alt="Screenshot 2023-08-29 at 10 33 48" src="https://github.com/zcash/secant-ios-wallet/assets/7198042/a8dc1953-0f48-489d-a7b3-d5df8972c9f2">

so I guess xcode had one of those days... I fixed the indentation on a source file level.